snp_motion_planning/launch/planning_server.launch.xml

```diff
@@ -22,6 +22,7 @@
     <param name="task_name" value="$(var task_name)"/>
     <param name="octree_resolution" value="0.010"/>
     <param name="collision_object_type" value="$(var collision_object_type)"/>
+    <param name="tcp_max_speed" value = "0.25"/>
```

Comment:
Set this param up so it can be changed by a launch arg, so we can set it from the automate_22 package start.launch